edx_proctoring/api.py

```diff
@@ -553,25 +553,14 @@ def create_exam_attempt(exam_id, user_id, taking_as_proctored=False):
 
             raise StudentExamAttemptAlreadyExistsException(err_msg)
 
-    allowed_time_limit_mins = exam['time_limit_mins']
-
-    # add in the allowed additional time
-    allowance_extra_mins = ProctoredExamStudentAllowance.get_additional_time_granted(exam_id, user_id)
-    if allowance_extra_mins:
-        allowed_time_limit_mins += allowance_extra_mins
-
-    allowed_time_limit_mins, is_exam_past_due_date = _calculate_allowed_mins(
-        exam['due_date'],
-        allowed_time_limit_mins
-    )
-
+    allowed_time_limit_mins = _calculate_allowed_mins(exam_id, user_id)
```

Comment:
can we pass `exam` here to avoid fetching it from db again? 

e.g. to avoid `get_exam_by_id(exam_id)`

Reply:
We are passing `allowed_time_limit_mins ` in context at L#585, will that start the timer for the learner?

Reply:
`allowed_time_limit_mins ` passed in context is for third party software. We pass this time at time of registering exam with proctoring software.

Reply:
@efischer19 do you have any idea why we pass `allowed_time_limit_mins` to proctoring software while registering exam attempt?